python/setup.py

```diff
@@ -201,7 +201,7 @@ def _supports_symlinks():
         extras_require={
             'ml': ['numpy>=1.7'],
             'mllib': ['numpy>=1.7'],
-            'sql': ['pandas>=0.13.0']
+            'sql': ['pandas>=0.19.2']
```

Comment:
Document this requirement and behavior changes in `Migration Guide`?

Reply:
Sure, I'll add it.